src/scripts/components/main/globe/globe.tsx

```diff
@@ -68,19 +79,23 @@ function getBasemapUrl(id: BasemapId | null) {
   return isElectron() ? config.basemapUrlsOffline[id] : config.basemapUrls[id];
 }
 
+const spinningEventHelper = new EventHelper();
```

Comment:
Why do we need this? Can't we just attach the handler directly to the clock?
And if we need it, shouldn't it be scoped inside the component so that multiple globe instances don't share the same object